<!DOCTYPE html>
<html lang="zh-cn">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <title> - 00002000 blog</title>
  <meta name="renderer" content="webkit" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>

<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />

<meta name="theme-color" content="#f8f5ec" />
<meta name="msapplication-navbutton-color" content="#f8f5ec">
<meta name="apple-mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-status-bar-style" content="#f8f5ec">


<meta name="author" content="00002000" /><meta name="description" content="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 21 aur/kalign 2.0.4-2 (&#43;0 0.00) (Orphaned) A fast and accurate multiple sequence alignment algorithm designed to align large numbers of sequences 20 aur/pfam 31.0-1 (&#43;0 0.00) The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs)." /><meta name="keywords" content="blog, 00002000, æŠ€æœ¯" />






<meta name="generator" content="Hugo 0.118.2 with theme even" />


<link rel="canonical" href="https://blog.00002000.xyz/post/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/sequence-alignment/" />
<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/manifest.json">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">



<link href="/sass/main.min.f92fd13721ddf72129410fd8250e73152cc6f2438082b6c0208dc24ee7c13fc4.css" rel="stylesheet">
<link href="/lib/fancybox/jquery.fancybox-3.1.20.min.css" rel="stylesheet">


<meta property="og:title" content="" />
<meta property="og:description" content="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 21 aur/kalign 2.0.4-2 (&#43;0 0.00) (Orphaned) A fast and accurate multiple sequence alignment algorithm designed to align large numbers of sequences 20 aur/pfam 31.0-1 (&#43;0 0.00) The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://blog.00002000.xyz/post/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/sequence-alignment/" /><meta property="article:section" content="post" />


<meta itemprop="name" content="">
<meta itemprop="description" content="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 21 aur/kalign 2.0.4-2 (&#43;0 0.00) (Orphaned) A fast and accurate multiple sequence alignment algorithm designed to align large numbers of sequences 20 aur/pfam 31.0-1 (&#43;0 0.00) The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs).">

<meta itemprop="wordCount" content="436">
<meta itemprop="keywords" content="" /><meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content=""/>
<meta name="twitter:description" content="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 21 aur/kalign 2.0.4-2 (&#43;0 0.00) (Orphaned) A fast and accurate multiple sequence alignment algorithm designed to align large numbers of sequences 20 aur/pfam 31.0-1 (&#43;0 0.00) The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs)."/>

<!--[if lte IE 9]>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/classlist/1.1.20170427/classList.min.js"></script>
<![endif]-->

<!--[if lt IE 9]>
  <script src="https://cdn.jsdelivr.net/npm/html5shiv@3.7.3/dist/html5shiv.min.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/respond.js@1.4.2/dest/respond.min.js"></script>
<![endif]-->

</head>
<body>
  <div id="mobile-navbar" class="mobile-navbar">
  <div class="mobile-header-logo">
    <a href="/" class="logo">00002000</a>
  </div>
  <div class="mobile-navbar-icon">
    <span></span>
    <span></span>
    <span></span>
  </div>
</div>
<nav id="mobile-menu" class="mobile-menu slideout-menu">
  <ul class="mobile-menu-list">
    <a href="/">
        <li class="mobile-menu-item">ä¸»é¡µ</li>
      </a><a href="/post/">
        <li class="mobile-menu-item">å½’æ¡£</li>
      </a><a href="/tags/">
        <li class="mobile-menu-item">æ ‡ç­¾</li>
      </a><a href="/categories/">
        <li class="mobile-menu-item">åˆé›†</li>
      </a><a href="/about/">
        <li class="mobile-menu-item">å…³äºæˆ‘</li>
      </a>
  </ul>

  


</nav>

  <div class="container" id="mobile-panel">
    <header id="header" class="header">
        <div class="logo-wrapper">
  <a href="/" class="logo">00002000</a>
</div>





<nav class="site-navbar">
  <ul id="menu" class="menu">
    <li class="menu-item">
        <a class="menu-item-link" href="/">ä¸»é¡µ</a>
      </li><li class="menu-item">
        <a class="menu-item-link" href="/post/">å½’æ¡£</a>
      </li><li class="menu-item">
        <a class="menu-item-link" href="/tags/">æ ‡ç­¾</a>
      </li><li class="menu-item">
        <a class="menu-item-link" href="/categories/">åˆé›†</a>
      </li><li class="menu-item">
        <a class="menu-item-link" href="/about/">å…³äºæˆ‘</a>
      </li>
  </ul>
</nav>

    </header>

    <main id="main" class="main">
      <div class="content-wrapper">
        <div id="content" class="content">
          <article class="post">
    
    <header class="post-header">
      <h1 class="post-title"></h1>

      <div class="post-meta">
        <span class="post-time"> 0001-01-01 </span>
        
        
      </div>
    </header>

    <div class="post-toc" id="post-toc">
  <h2 class="post-toc-title">æ–‡ç« ç›®å½•</h2>
  <div class="post-toc-content always-active">
    <nav id="TableOfContents"></nav>
  </div>
</div>
  <div class="post-outdated">
    <div class="warn">
      <p>ã€æ³¨æ„ã€‘æœ€åæ›´æ–°äº <span class="timeago" datetime="0001-01-01T00:00:00" title="January 1, 0001">January 1, 0001</span>ï¼Œæ–‡ä¸­å†…å®¹å¯èƒ½å·²è¿‡æ—¶ï¼Œè¯·è°¨æ…ä½¿ç”¨ã€‚</p>
    </div>
  </div>
    <div class="post-content">
      <div class="highlight"><div class="chroma">
<table class="lntable"><tr><td class="lntd">
<pre tabindex="0" class="chroma"><code><span class="lnt"> 1
</span><span class="lnt"> 2
</span><span class="lnt"> 3
</span><span class="lnt"> 4
</span><span class="lnt"> 5
</span><span class="lnt"> 6
</span><span class="lnt"> 7
</span><span class="lnt"> 8
</span><span class="lnt"> 9
</span><span class="lnt">10
</span><span class="lnt">11
</span><span class="lnt">12
</span><span class="lnt">13
</span><span class="lnt">14
</span><span class="lnt">15
</span><span class="lnt">16
</span><span class="lnt">17
</span><span class="lnt">18
</span><span class="lnt">19
</span><span class="lnt">20
</span><span class="lnt">21
</span><span class="lnt">22
</span><span class="lnt">23
</span><span class="lnt">24
</span><span class="lnt">25
</span><span class="lnt">26
</span><span class="lnt">27
</span><span class="lnt">28
</span><span class="lnt">29
</span><span class="lnt">30
</span><span class="lnt">31
</span><span class="lnt">32
</span><span class="lnt">33
</span><span class="lnt">34
</span></code></pre></td>
<td class="lntd">
<pre tabindex="0" class="chroma"><code class="language-fallback" data-lang="fallback"><span class="line"><span class="cl">21 aur/kalign 2.0.4-2 (+0 0.00) (Orphaned) 
</span></span><span class="line"><span class="cl">    A fast and accurate multiple sequence alignment algorithm designed to align large numbers of sequences
</span></span><span class="line"><span class="cl">20 aur/pfam 31.0-1 (+0 0.00) 
</span></span><span class="line"><span class="cl">    The Pfam database is a large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs).
</span></span><span class="line"><span class="cl">19 aur/gotohscan 2.0_alpha-1 (+0 0.00) 
</span></span><span class="line"><span class="cl">    Search short sequences in large database sequences by computing all semi-global alignments
</span></span><span class="line"><span class="cl">18 aur/msaprobs 0.9.7-1 (+0 0.00) (Orphaned) 
</span></span><span class="line"><span class="cl">    Multiple sequence alignment generator
</span></span><span class="line"><span class="cl">17 aur/bamrescue 0.2.0-1 (+0 0.00) 
</span></span><span class="line"><span class="cl">    Utility to check Binary Sequence Alignment / Map (BAM) files for corruption and repair them
</span></span><span class="line"><span class="cl">16 aur/tigrfam 15.0-1 (+0 0.00) 
</span></span><span class="line"><span class="cl">    TIGRFAMs is a resource consisting of curated multiple sequence alignments, Hidden Markov Models (HMMs) for protein sequence classification, and associated information designed to support automated annotation of (mostly prokaryotic) proteins.
</span></span><span class="line"><span class="cl">15 aur/dialign 2.2.1-3 (+1 0.00) 
</span></span><span class="line"><span class="cl">    Multiple sequence alignment program
</span></span><span class="line"><span class="cl">14 aur/gblocks-bin 0.91b-1 (+1 0.00) 
</span></span><span class="line"><span class="cl">    A program written in ANSI C language that eliminates poorly aligned positions and divergent regions of an alignment of DNA or protein sequences
</span></span><span class="line"><span class="cl">13 aur/gmap-gsnap 2019.03.15-1 (+3 0.00) (Orphaned) (Out-of-date: 2022-04-24) 
</span></span><span class="line"><span class="cl">    A Genomic Mapping and Alignment Program for mRNA and EST Sequences, and Genomic Short-read Nucleotide Alignment Program
</span></span><span class="line"><span class="cl">12 aur/clustalx 2.1-6 (+3 0.00) (Installed: 2.1-2)
</span></span><span class="line"><span class="cl">    Multiple alignment of nucleic acid and protein sequences
</span></span><span class="line"><span class="cl">11 aur/clustalw 2.1-4 (+4 0.00) (Installed: 2.1-2)
</span></span><span class="line"><span class="cl">    Clustal W multiple sequence alignment program, version 2.0
</span></span><span class="line"><span class="cl">10 aur/tcoffee 13.45.0.4846264-1 (+6 0.00) 
</span></span><span class="line"><span class="cl">    An alignment tool for Protein, DNA and RNA sequences
</span></span><span class="line"><span class="cl">9 aur/bowtie 1.3.1-1 (+8 0.00) 
</span></span><span class="line"><span class="cl">    Alignment tool for short nucleotide sequences against long templates
</span></span><span class="line"><span class="cl">8 aur/seaview 5.0.4-2 (+10 0.00) 
</span></span><span class="line"><span class="cl">    GUI for multiple sequence alignment and molecular phylogeny
</span></span><span class="line"><span class="cl">7 aur/python-pysam 0.18.0-1 (+11 0.00) 
</span></span><span class="line"><span class="cl">    Python interface for the SAM/BAM sequence alignment and mapping format
</span></span><span class="line"><span class="cl">6 aur/clustal-omega 1.2.4-1 (+12 0.00) 
</span></span><span class="line"><span class="cl">    Protein sequence alignment program
</span></span><span class="line"><span class="cl">5 aur/mafft 7.490-1 (+13 0.00) (Installed)
</span></span><span class="line"><span class="cl">    Multiple alignment program for amino acid or nucleotide sequences
</span></span></code></pre></td></tr></table>
</div>
</div><p>4 archlinuxcn/mafft 7.490-1 (1.3 MiB 13.2 MiB) (Installed)
Multiple alignment program for amino acid or nucleotide sequences
3 archlinuxcn/clustalx 2.1-2 (476.7 KiB 1.4 MiB) (Installed)
Multiple alignment of nucleic acid and protein sequences (command line tool
2 archlinuxcn/clustalw 2.1-2 (280.6 KiB 725.1 KiB) (Installed)
Multiple alignment of nucleic acid and protein sequences (command line tool)
1 archlinuxcn/clustal-omega 1.2.4-2 (413.4 KiB 1.8 MiB)
A general purpose multiple sequence alignment program for protein and DNA/RNA
2 aur/gblocks-bin 0.91b-1 (+1 0.00)
A program written in ANSI C language that eliminates poorly aligned positions and divergent regions of an alignment of DNA or protein sequences</p>

      <details><summary>End</summary>is start.</details>
    </div>

    
<div class="post-reward">
  <input type="checkbox" name="reward" id="reward" hidden />
  <label class="reward-button" for="reward">èµèµæ”¯æŒ</label>
  <div class="qr-code">
    
    <label class="qr-code-image" for="reward">
        <img class="image" src="/img/reward/wechat-pay.jpg">
        <span>å¾®ä¿¡æ‰“èµ</span>
      </label>
    <label class="qr-code-image" for="reward">
        <img class="image" src="/img/reward/alipay.jpg">
        <span>æ”¯ä»˜å®æ‰“èµ</span>
      </label>
  </div>
</div><footer class="post-footer">
      
      <nav class="post-nav">
        <a class="prev" href="/post/linux/megax/">
            <i class="iconfont icon-left"></i>
            <span class="prev-text nav-default"></span>
            <span class="prev-text nav-mobile">ä¸Šä¸€ç¯‡</span>
          </a>
        
      </nav>
    </footer>
  </article>
        </div>
        

  

  


    <div>
      <div class="pagination__title">
        <span class="pagination__title-h" style="font-size: 20px;">ğŸ’¬è¯„è®º</span>
      <hr/>
    </div>
  <div id="tcomment"></div>
    <script src="https://cdn.staticfile.org/twikoo/1.6.20/twikoo.all.min.js">
    </script>
    <script>
        twikoo.init({
            envId: "00002000",
        el: "#tcomment",
            lang: 'zh-CN',
            region: "ap-guangzhou",
        path: window.TWIKOO_MAGIC_PATH||window.location.pathname,
        })
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://github.com/utterance">comments powered by Twikoo.</a></noscript>

      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="social-links">
      <a href="mailto:your@email.com" class="iconfont icon-email" title="email"></a>
      <a href="http://localhost:1313" class="iconfont icon-stack-overflow" title="stack-overflow"></a>
      <a href="http://localhost:1313" class="iconfont icon-twitter" title="twitter"></a>
      <a href="http://localhost:1313" class="iconfont icon-facebook" title="facebook"></a>
      <a href="http://localhost:1313" class="iconfont icon-linkedin" title="linkedin"></a>
      <a href="http://localhost:1313" class="iconfont icon-google" title="google"></a>
      <a href="http://localhost:1313" class="iconfont icon-github" title="github"></a>
      <a href="http://localhost:1313" class="iconfont icon-weibo" title="weibo"></a>
      <a href="http://localhost:1313" class="iconfont icon-zhihu" title="zhihu"></a>
      <a href="http://localhost:1313" class="iconfont icon-douban" title="douban"></a>
      <a href="http://localhost:1313" class="iconfont icon-pocket" title="pocket"></a>
      <a href="http://localhost:1313" class="iconfont icon-tumblr" title="tumblr"></a>
      <a href="http://localhost:1313" class="iconfont icon-instagram" title="instagram"></a>
      <a href="http://localhost:1313" class="iconfont icon-gitlab" title="gitlab"></a>
      <a href="http://localhost:1313" class="iconfont icon-bilibili" title="bilibili"></a>
    <a href="https://blog.00002000.xyz/index.xml" type="application/rss+xml" class="iconfont icon-rss" title="rss"></a>
</div>

<div class="copyright">
  <span class="power-by">
    ç”± <a class="hexo-link" href="https://gohugo.io">Hugo</a> å¼ºåŠ›é©±åŠ¨
  </span>
  <span class="division">|</span>
  <span class="theme-info">
    ä¸»é¢˜ - 
    <a class="theme-link" href="https://github.com/olOwOlo/hugo-theme-even">Even</a>
  </span>

  

  <span class="copyright-year">
    &copy; 
    2017 - 
    2023<span class="heart"><i class="iconfont icon-heart"></i></span><span>00002000</span>
  </span>
</div>

    </footer>

    <div class="back-to-top" id="back-to-top">
      <i class="iconfont icon-up"></i>
    </div>
  </div>
  <script type="text/javascript" src="/lib/jquery/jquery-3.2.1.min.js"></script>
  <script type="text/javascript" src="/lib/slideout/slideout-1.0.1.min.js"></script>
  <script type="text/javascript" src="/lib/fancybox/jquery.fancybox-3.1.20.min.js"></script>
    <script type="text/javascript" src="/lib/timeago/timeago-3.0.2.min.js"></script>
    <script type="text/javascript" src="/lib/timeago/timeago.locales-3.0.2.min.js"></script>
  <script><!-- NOTE: timeago.js uses the language code format like "zh_CN" (underscore and case sensitive) -->
    var languageCode = "zh_CN".replace(/-/g, '_').replace(/_(.*)/, function ($0, $1) {return $0.replace($1, $1.toUpperCase());});
    timeago().render(document.querySelectorAll('.timeago'), languageCode);
    timeago.cancel();  
  </script>



<script type="text/javascript" src="/js/main.min.4ae89da218555efa0e7093a20b92017d2e1202b66fff9fc2edf4cb8d44b44c6e.js"></script>








</body>
</html>
